### PR TITLE
Added support for logging integration via ZConfig:

### PR DESCRIPTION
**Pull request by jimfulton**
See: http://zconfig.readthedocs.io/en/latest/using-logging.html

Note that the Zope integration also uses ZConfig, but adds lots of Zope-specific machinery that won't work elsewhere.  This integration cribbed from the Zope integration. :)

### CODE REVIEW

**Comment by jimfulton** [comment]
I've cleaned up the implementation a little and added tests.

File: docs/integrations/zconfig.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
+ZConfig logging configuration
+=============================
+
+`ZConfig
+<http://zconfig.readthedocs.io/en/latest/using-logging.html>`_
+provides one of the most powerful clean configuration mechanism for
```
Comment (dcramer) [inline docs/integrations/zconfig.rst]: it probably makes sense to keep the description unbiased:

> ZConfig provides a configuration mechanism for the Python logging module

**Comment by jimfulton** [comment]
OK. language changed.

**Comment by dcramer** [comment]
thanks!

**Pull request merged**